ClaimOwnership/types.ts

```diff
@@ -0,0 +1,12 @@
+import {
+  NonEmptyString,
+  OrganizationFiscalCode
+} from "@pagopa/ts-commons/lib/strings";
+import * as t from "io-ts";
+
+// The shape of the item expected in the queue
+export type OrganizationQueueItem = t.TypeOf<typeof OrganizationQueueItem>;
```

Comment:
Naming: imho `OrganizationQueueItem` doesn't communicate enough what the item is for. Isn't that a `Claim` "entity" (defined by the tuple Organization-Delegate)?

Reply:
Refactor with 79f7545